maistra/scripts/update-deps.sh

```diff
@@ -83,7 +83,7 @@ function copy_files() {
         cp_flags="-r"
       fi
       cp "${cp_flags}" "${f}" "${VENDOR_DIR}" || echo "Copy of ${f} failed. Ignoring..."
-      echo "build --override_repository=${repo_name}=/work/maistra/vendor/${repo_name}" >> "${BAZELRC}"
+      echo "build --override_repository=${repo_name}=%workspace%/maistra/vendor/${repo_name}" >> "${BAZELRC}"
```

Comment:
Any particular reason for this change? This will break downstream build, as it relies on /work.

Reply:
It's a bit awkward for those who do not mount the workspace as `/work` (I mount mine to the same path in the container as on the host, which I find a lot easier). This change just made the scripts work regardless of where the workspace is mounted into the container

Reply:
Removed the use of `%workspace%` and reinstated `/work`